scripts/electron-node/requireHook.js

```diff
@@ -0,0 +1,5 @@
+const flag = "--ms-enable-electron-run-as-node";

```

Comment:
VS Code requiring this flag really screws with things, hence this terrible hook.